### PR TITLE
fix file-store api

### PR DESCRIPTION
**Pull request by JameelB**
## Description
Fixed and added securityMiddleware to the filestore API\

NOTE: Endpoint for filestore has been changed to `/file/binary/` or `/file/base64/`

### CODE REVIEW

**Review by wtrocki — APPROVED** [review]
Verified. Works like dream!

**Comment by wtrocki** [comment]
@JameelB Going to adjust client to new API.

File: cloud/filestore/src/FileRoutes.ts [hunk 1]
```diff
@@ -11,8 +11,9 @@ import * as fileService from './services/FileService';
  * @returns router instance of express router
  */
 export function createRouter(storageEngine: FileStorage) {
+  fileService.createTemporaryStorageFolder();
   const router = Router();
-  router.route('/:filename').post(function(req, res, next) {
+  router.route('/base64/:filename').post(function(req, res, next) {
```
Comment (paolobueno) [inline cloud/filestore/src/FileRoutes.ts:16]: Do you guys know the exact format needed for `parseBase64Stream`? Guess we can just test it out once this is merged :+1:

Comment (wtrocki) [inline cloud/filestore/src/FileRoutes.ts:16]: Looks like filename is not used/needed. We can remove it in separate PR.

**Pull request merged**